validation.js

```diff
@@ -0,0 +1,22 @@
+const form = document.getElementById('contact-form');
+const email = document.getElementById('email');
+const error = document.getElementById('error-msg');
+
+form.addEventListener('submit', (event) => {
+  const errArray = [];
+  
+  if(email.value === '') {
+    errArray.push('Invalid email')
+  } else if(email.value !== email.value.toLowerCase()) {
+    errArray.push('Invalid email, ensure characters are in lowerCase')
+  } else {
+    form.submit();
+    form.reset();
+  }
+  
+  if(errArray.length > 0) {
+    event.preventDefault();
+    error.innerHTML = errArray.join(', ')
+  }
+});
```

Comment:
well done so far @yegonkimutai, but I would suggest that you attach this eventLisner to the click event of the button and prevent the default behavior of submitting the from and submit after validation 
`Button.addEventListener('click', (e) => {
  e.preventDefault();
  do validation ...
  form.submit()
`